### DISCUSSION TITLE
The "expanded" hover state in the new beatmap cards can disappear randomly

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#16085] [Q&A] [answered]
Sometimes the "expanded" hover state of the new beatmap cards disappears, even though I dragged the cursor only to the area where it should remain active.
Unfortunately, I'm not sure how to reproduce it easily, but it seems to happen only when the cursor is on the expanded difficulty list.

https://user-images.githubusercontent.com/43073074/146045052-14bffb2e-a5b5-448a-958a-21084209fbe0.mp4

**Comment by Gabixel** [comment]
Not sure if I did something similar in the previous video, but an easy way to reproduce it very often seems to be when you move the cursor to another card (in a short time).
It won't happen 100% of the time, but these steps seem to have something to do with the issue.

https://user-images.githubusercontent.com/43073074/146056519-e69f8c8a-90de-4a17-846b-176313092e63.mp4

**Reply by bdach** [reply]
have spent about 10 minutes trying to reproduce this manually following the video above and couldn't.

**Reply by Gabixel** [reply]
Looks like the reason is different:
I'm not sure if there are multiple ways (since in the first video sent I managed to reproduce the issue in some other way).
The easiest one happens when the expanded list is fading in and, while it's fading, you rapidly move the cursor to another beatmap card to extend it (in the area where there's the rank status):

https://user-images.githubusercontent.com/43073074/146064679-ce3d039d-3155-4dba-97b8-461b0a6988e6.mp4

(I guess it's not a bug, but surely it's a very rare case where the user experience could suffer from it)

**Reply by bdach** [reply]
i *think* i can see how this can happen. probably has to do with the hide logic, i'm thinking it is sometimes somehow possible to trigger both an immediate hide and a delayed hide in the same frame or something, and the second re-fires after re-hovering the card again..?

**Comment by peppy** [comment, marked as answer]
Will be resolved via https://github.com/ppy/osu/pull/16094.
